releases/stellar/2025-01-ITS-v1.0.0.md

```diff
@@ -0,0 +1,149 @@
+- [Releases](https://github.com/axelarnetwork/axelar-cgp-stellar/releases)
+
+## Background
+
+Changes in the release:
+
+1. This is the v1.0.0 initial ITS release.
+
+## Deployment
+
+Create an `.env` config.
+
+```yaml
+PRIVATE_KEY=xyz
+ENV=xyz
+CHAIN=stellar
+```
+
+- ITS depends on GMP to function, so ensure that GMP is deployed first.
+
+```bash
+# Deploy Interchain Token
+[1] node stellar/deploy-contract.js deploy interchain_token --version v1.0.0
+
+# Deploy Token Manager
+[2] node stellar/deploy-contract.js deploy token_manager --version v1.0.0
+
+# Deploy Interchain Token Service
+[3] node stellar/deploy-contract.js deploy interchain_token_service --version v1.0.0
+
+# Deploy Example
+[4] node stellar/deploy-contract.js deploy example --version v1.0.0
+```
+
+### Setting up trusted chains from Stellar
+
+```bash
+[5] node stellar/its.js add-trusted-chains axelar
+[6] node stellar/its.js add-trusted-chains all
+```
+
+### Setting up trusted chains from EVM
+
+- Change private key and env from stellar to EVM
+
+```bash
+[7] node evm/its.js -n all --action setTrustedAddress --trustedChain stellar --trustedAddress hub
+```
+
+## Checklist
+
+The following checks should be performed after the rollout
+
+```bash
+ITS_HUB_ADDRESS=its_hub_addres
+STELLAR_TOKEN_ADDRESS=CDLZFC3SYJYDZT7K67VZ75HPJVIEUVNIXF47ZG2FB2RMQQVU2HHGCYSC # Testnet
+#STELLAR_TOKEN_ADDRESS=CAS3J7GYLGXMF6TDJBBYYSE3HQ6BBSMLNUQ34T6TZMYMW2EVH34XOWMA # Mainnet
+```
+
+### Stellar to EVM
+
+- [ ] Deploy Native Interchain Token
+
+```bash
+node stellar/its.js deploy-interchain-token test test 18 0x1234 10000
+
+node stellar/its.js deploy-remote-interchain-token 0x1234 avalanche $STELLAR_TOKEN_ADDRESS 1
+
+node evm/gateway.js -n avalanche --action execute --payload $PAYLOAD --sourceChain axelar --sourceAddress $ITS_HUB_ADDRESS --messageId 0x1c297d6a2522aa9263d4ac034252486284d58f0c9badf1f0beb009b27c37ab1f-298 --destination 0x549Ec0Df306248e2AbDa6e71a0C0aDf3c9FB7886
+```
+
+- [ ] Interchain Token Transfer for Native Interchain Token
+
+```bash
+node stellar/its.js interchain-transfer 0xa98974cd02e5ee3d65bff69465a00917f27176bffd72e352c11a78c7a999bded avalanche 0x549Ec0Df306248e2AbDa6e71a0C0aDf3c9FB7886 1 0x1234 $STELLAR_TOKEN_ADDRESS 1
+
+node evm/gateway.js -n avalanche --action execute --payload $PAYLOAD --sourceChain axelar --sourceAddress $ITS_HUB_ADDRESS --messageId 0x1d47435f80dd8c7cdfacd94ec8834b346b354f0087c9b8a305498fdd2b2bd3c2-343 --destination 0x549Ec0Df306248e2AbDa6e71a0C0aDf3c9FB7886
+```
+
+- [ ] Deploy Remote Canonical Token
+
+```bash
+node stellar/its.js register-canonical-token $STELLAR_TOKEN_ADDRESS
+
+node stellar/its.js deploy-remote-canonical-token $STELLAR_TOKEN_ADDRESS avalanche $STELLAR_TOKEN_ADDRESS 1
+
+node evm/gateway.js -n avalanche --action execute --payload $PAYLOAD --sourceChain axelar --sourceAddress $ITS_HUB_ADDRESS --messageId 0xd8adcfdd262da322ac5350704ee50afb0a944e5bb08c9e887f11a7528e157de9-461 --destination 0x549Ec0Df306248e2AbDa6e71a0C0aDf3c9FB7886
```

Comment:
avoid hardcoding addresses, create a `$WALLET` to reuse deployer address if needed